packages/serverless/src/model/abstract/document.ts

```diff
@@ -302,7 +143,7 @@ possible field kinds explicitly to ensure correctness.`
     return result;
   }
 
-  public async findOne(filter: Filter<any>, session?: ClientSession) {
+  public async findOneActive(filter: Filter<any>, session?: ClientSession) {
```

Comment:
try avoid any as much as possible this method we can use type

Reply:
fixed